src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/IconFieldTransformer.js

```diff
@@ -17,8 +17,10 @@ export default class IconFieldTransformer implements FieldTransformer {
 
         const {
             mapping,
+            default_mapping,
             skin = 'default',
         }: {
+            default_mapping: Object | string,
```

Comment:
I would go with  `default` instead of `default_mapping` or `default_icon`, mapping sounds for me strange here. @nnatter what do you think?

Reply:
> I would go with `default` instead of `default_mapping` or `default_icon`, mapping sounds for me strange here. @nnatter what do you think?

At first I wanted to use `default_value` like we have it in the content-types (`single_select` or `checkbox`). But it felt kinda weird to have a `default_value` which is the `default` for the `mapping` collection. Just using `default` is also be fine by me.